react-app/src/components/UserCard/UserCard.tsx

```diff
@@ -0,0 +1,27 @@
+import { UserFormData } from '../../types/types';
+import './UserCard.scss';
+
+type UserCardProps = {
+  form: UserFormData;
+};
```

Comment:
and no need for unpacking form and then its inner fields
```suggestion
type UserCardProps = UserFormData;
```